visualization/Readme.md

```diff
@@ -0,0 +1,32 @@
+This is a tool for data analysis. It has two pages: "Run model" and "Analyze".
+## Run Prompt page:
+This page allows for the analysis of model answers depending on different parameters we set up. It has three modes: "Complete", "Run one test", and "Run whole dataset".
+**Complete** mode allows for a conversation with the model. It has the minimum parameters to set up.
```

Comment:
Should this be split into multiple lines? You can use some tool (e.g. vs-code) to preview the markdown, right now this is all just one really big line